RealmBrowserTests/Supporting Files/RealmBrowserTests-Info.plist

```diff
@@ -13,10 +13,10 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.1.8</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>92</string>
```

Comment:
I started using agvtool (inside fastlane) which modifies version number for all targets in projects. Actually having the same version for tests target seems absolutely fine.